Rlabkey/man/labkey.domain.create.Rd

```diff
@@ -52,6 +52,7 @@ to the domain kind specified. The list of valid options for each domain kind are
   	        \item{ \code{demographics} : (TRUE | FALSE) Determines whether the dataset is created as demographic}
   	        \item{ \code{keyPropertyName} : The name of an additional key field to be used in conjunction with participantId and (visitId or date) to create unique records}
   	        \item{ \code{useTimeKeyField} : (TRUE | FALSE) Specifies to use the time portion of the date field as an additional key }
+  	        \item{ \code{isManagedField} : (TRUE | FALSE) Specifies whether the field from keyPropertyName should be managed by LabKey. }
```

Comment:
So this is already supported on the server side right? I like the change that just require a doc update to use the "feature".